packages/core/src/RenderingEngine/StackViewport.ts

```diff
@@ -456,7 +448,9 @@ class StackViewport extends Viewport implements IStackViewport, IImagesLoader {
       spacing: vtkImageData.getSpacing(),
       origin: vtkImageData.getOrigin(),
       direction: vtkImageData.getDirection(),
-      scalarData: vtkImageData.getPointData().getScalars().getData(),
+      get scalarData() {
+        return this.csImage.voxelManager.getScalarData();
```

Comment:
this.csImage?.voxelManager.getScalarData()
as csImage can be null if things aren't yet setup, and this shouldn't throw NPE